ui/app/styles/core/charts.scss

```diff
@@ -74,13 +74,6 @@
   }
 }
 
-.chart {
```

Comment:
Just realize I'm going to need to keep this for my chart. But `.axes-lines` CSS can be used for the line chart, too